go.mod

```diff
@@ -4,39 +4,38 @@ go 1.20
 
 replace (
 	github.com/knative/pkg => github.com/rancher/pkg v0.0.0-20190514055449-b30ab9de040e
-	k8s.io/client-go => k8s.io/client-go v0.27.4
 	sigs.k8s.io/json => sigs.k8s.io/json v0.0.0-20211208200746-9f7c6b3444d2
 )
 
 require (
-	github.com/Masterminds/sprig/v3 v3.2.3
-	github.com/apparentlymart/go-cidr v1.1.0
-	github.com/aws/aws-sdk-go v1.44.272
+	github.com/Masterminds/sprig/v3 v3.2.2
+	github.com/apparentlymart/go-cidr v1.0.1
+	github.com/aws/aws-sdk-go v1.38.65
```

Comment:
While these dependencies look like they're downgraded due to docker change, the net diff for v1.5.0 from v1.4.x is the same: 
```
github.com/Masterminds/sprig/v3 v3.2.2 => v3.2.2
github.com/apparentlymart/go-cidr v1.0.1 => v1.0.1
github.com/aws/aws-sdk-go v1.38.65 => v1.38.65
```